libcxx/utils/generate_feature_test_macro_components.py

```diff
@@ -415,8 +415,11 @@ def add_version_header(tc):
         },
         {
             "name": "__cpp_lib_constrained_equality",
-            "values": {"c++26": 202403}, # P2944R3: Comparisons for reference_wrapper
-            "headers": ["optional", "tuple", "utility", "variant"],
+            "values": {
+                # "c++26": 202403,  # P2944R3: Comparisons for reference_wrapper
+                "c++26": 202411,  # P3379R0: Constrain std::expected equality operators
+            },
+            "headers": ["expected", "optional", "tuple", "utility", "variant"],
```

Comment:
It seems better to me no to touch the FTM until having implemented the whole P2944R3.

Reply:
This doesn't set the FTM it just adds the new value to the script.

Reply:
I have this as a separate subtask. We also need to implement the tuple-like overload I believe before we can set the FTM. BTW I have WIP branch for the remaining bits of P2944R3.